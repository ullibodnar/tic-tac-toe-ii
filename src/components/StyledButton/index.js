import styled from 'styled-components'

const StyledButton = styled.button`
  border-radius: 0;
  padding: 10px 15px;
  font-size: 22px;
  background: #191919;
  color: #e2e1e0;
  border: none;
  box-shadow: 0 10px 20px rgba(0,0,0,0.19), 0 6px 6px rgba(0,0,0,0.23);
  transition: all 0.3s cubic-bezier(.25,.8,.25,1);
  margin-bottom: 20px;

  &:hover {
    box-shadow: 0 14px 28px rgba(0,0,0,0.25), 0 10px 10px rgba(0,0,0,0.22);
    cursor: pointer;
    
  }

  &:focus {
    outline: none;
  }
`
StyledButton.defaultName = 'StyledButton'

export default StyledButton
