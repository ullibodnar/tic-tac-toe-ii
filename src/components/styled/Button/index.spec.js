import React from 'react'
import { shallow } from 'enzyme'

import StyledButton from '.'

describe('components:StyledButton', () => {
  it('renders the button with the correct styles', () => {
    expect(toJson(shallow(<StyledButton />))).toMatchSnapshot()
  })
})
