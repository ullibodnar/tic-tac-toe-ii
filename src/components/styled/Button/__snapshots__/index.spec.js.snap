// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components:StyledButton renders the button with the correct styles 1`] = `
.c0 {
  border-radius: 0;
  padding: 10px 15px;
  font-size: 22px;
  background: #191919;
  color: white;
  border: none;
  box-shadow: 0 10px 20px rgba(0,0,0,0.19),0 6px 6px rgba(0,0,0,0.23);
  -webkit-transition: all 0.3s cubic-bezier(.25,.8,.25,1);
  transition: all 0.3s cubic-bezier(.25,.8,.25,1);
  margin-bottom: 20px;
}

.c0:hover {
  box-shadow: 0 14px 28px rgba(0,0,0,0.25),0 10px 10px rgba(0,0,0,0.22);
  cursor: pointer;
}

.c0:focus {
  outline: none;
}

<button
  className="c0"
/>
`;
